backend/backend/services/openAI/service.py

```diff
@@ -1,43 +1,44 @@
 import openai
 
 from backend.config import OPENAI_API_KEY
-from backend.services.graph.graphql_examples import LIST_OF_EXAMPLES
 from backend.services.openAI.graph_prompt_factory import GraphPromptFactory
 from llama_index import LLMPredictor, GPTSimpleVectorIndex, SimpleDirectoryReader, PromptHelper
 from llama_index.indices import GPTListIndex
 from llama_index import Document
+from backend.services.graph.service import GraphService
 
 
 from langchain import OpenAI
 from backend.services.graph import subgraphs
 import os
 
+PROTOCOL_TO_PATH = {
+    "opensea-v2": "opensea",
+    "opensea-v1": "opensea"
+}
 
 
-def generate_prompt(input):
-    """
-    Take in query from user and prepend sample queries
-    """
-    sample_queries = LIST_OF_EXAMPLES
-    return (
-        sample_queries
-        + PRE_PROMPT.format(input)
-    )
+def protocol_path_formatter(protocol):
+    return PROTOCOL_TO_PATH.get(protocol, protocol)
+
+
+def mapping_path(protocol):
+    path = protocol_path_formatter(protocol)
```

Comment:
this is not needed. calling `graph_service.subgraph.deployments['base']` gives the folder location already.

so `graph_service.subgraph.deployments['base']` will give `opensea`, both for `graph_service.subgraph.deployments['protocol'] == "opensea-v1"` and `== "opensea-v2"`